src/happy.js

```diff
@@ -1 +1,2 @@
+//This is a program that doesn't take any input and logs the string literal "happy" to console
```

Comment:
```suggestion
// This is a program that logs the string literal "happy" to console.
```

Reply:
I agree with the whitespace at the beginning, however I really do believe that it's better to state the input of the program as another dev may be confused.

Reply:
Agreed. I edited the suggestion

Reply:
Thanks. Will try and make these changes when I get the time. 
ETA: a few days.